src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml

```diff
@@ -0,0 +1,104 @@
+﻿@using Esfa.Recruit.Vacancies.Client.Domain.Enums
+@model Esfa.Recruit.Employer.Web.ViewModels.Part1.Wage.WageViewModel
+
+<div class="grid-row">
+    <div class="column-full">
+
+        @Html.Partial("_Part1Header")
+
+        <div asp-validation-summary="All" class="error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+            <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
+                Please fix these errors
+            </h2>
+        </div>
+        
+        <form asp-route="RouteNames.Wages_Get">
+
+            <div class="form-group">
+                <fieldset>
+                    <label asp-for="Duration" class="form-label-bold">How long is the apprenticeship expected to last?</label>
+                    <span asp-validation-for="Duration" class="error-message"></span>
+                    <input asp-for="Duration" class="form-control form-control-1-8" type="number">
+                    <select asp-for="DurationUnit" class="form-control" style="width: auto">
+                        <option value="@DurationUnit.Month">@DurationUnit.Month.ToString()</option>
+                        <option value="@DurationUnit.Year">@DurationUnit.Year.ToString()</option>

```

Comment:
Could we display `Month(s)` and `Year(s)` here.

Reply:
👍